pkgs/top-level/all-packages.nix

```diff
@@ -14104,23 +14104,17 @@ let
 
   cuyo = callPackage ../games/cuyo { };
 
-  dfhack = callPackage_i686 ../games/dfhack {
-    inherit (pkgsi686Linux.perlPackages) XMLLibXML XMLLibXSLT;
-  };
-
   dhewm3 = callPackage ../games/dhewm3 {};
 
   drumkv1 = callPackage ../applications/audio/drumkv1 { };
 
-  dwarf_fortress = callPackage_i686 ../games/dwarf-fortress {
-    SDL_image = pkgsi686Linux.SDL_image.override {
-      libpng = pkgsi686Linux.libpng12;
-    };
-    inherit (pkgsi686Linux.perlPackages) XMLLibXML XMLLibXSLT;
+  dwarf-fortress-packages = callPackage ../games/dwarf-fortress { };
+
+  dwarf-fortress = dwarf-fortress-packages.dwarf-fortress.override {
     enableDFHack = config.dwarfFortress.enableDFHack or false;
```

Comment:
It feels this discussion would be long and the policy actually isn't as agreed upon as I've thought. I propose to resolve this as if the policy doesn't exist using your judgement as the current /de facto/ package maintainer. You would prefer the additional `config` variable to be added in this case, correct?